docs/git/desktop/lesson.md

```diff
@@ -396,29 +396,44 @@ Celebration!!!
 Make an effort to mark the achievements up to this point. You completed ITD and Fundamentals. Some of you never wrote a line of code before that!
 
 **Session 1: Goals and Expectations**
-The presentation below aims to identify your goal(s) for being on the CYF software development course; to highlight the expectations CodeYourFuture has for its trainees; to introduce the concept of progressing from the Intro to Digital, Fundamentals courses to the software development course.
 
-[Goals and Expectations](https://docs.google.com/presentation/d/11vimrefwpHfFByUfgPL-jwhZ2KrE0Li80M_O_z2U95I/edit)
+**Session objective:** The presentation below aims:
 
-Notes for the facilitator: Remember to adapt/update the content in line with your cohort’s needs and arrange timings as needed.
+- to identify your goal(s) for being on the CYF software development course
+- to highlight the expectations CodeYourFuture has for its trainees
+- to explain the progress from Intro to Digital to Fundamentals course, and then to the Software Development course.
+
+[Goals and Expectations](https://docs.google.com/presentation/d/154zub_LucLpwO_qThkbymArqDIfCfrMu40IDUvguq-I/edit#slide=id.gaa71b86f5e_0_0)
 
 **Session 2: What do developers do?**
-In this section we'll explore how a professional developer behaves and thus how you should behave.
 
-**Exercise (20 min)**
-In small groups. Imagine all the day-to-day activities and behaviours of a professional software developer. Use an online board (Miro, Google Jamboard, etc.). Think about whether you do all those things already. Discuss how to change things so you begin to do more and more of those things. Highlight that practice is the way to develop.
+**Session objective:** In this section we'll explore how a professional developer behaves and thus how you should behave.
+
+:::note Exercise (20 min)
+
+**Exercise objective:** To brainstorm and think about future responsibilities.
+
+In small groups, imagine all the day-to-day activities and behaviours of a professional software developer. Use a whiteboard, sticky notes, or an online board (Miro, Google Jamboard, etc.). Think about whether you do all those things already. Discuss how to change things so you begin to do more and more of those things. Highlight that practice is the way to develop.
```

Comment:
Previously in the page, the exercise heading is not in the note, and then the actual contents of the exercise is. Here, it's swapped - the exercise heading is a note, and the exercise itself is not. We should probably be consistent within the page about which we do - do you have a preference?

I have a weak preference for the way the top half of the page does it (because then the exercises show up in the navigation side-bar, because they're headings), but it's only a weak preference...

Reply:
Actually, looking at the rest of the syllabus, it looks like this way is the dominant way